### PR TITLE
Added Localization for Language

### PR DESCRIPTION
**Pull request by Artillio**
I Have added localization for language.
I Checked and all pages have the localization.

### CODE REVIEW

File: src/Server/appsettings.Development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(localdb)\\mssqllocaldb;Initial Catalog=BlazorHero.CleanArchitecture;Integrated Security=True;MultipleActiveResultSets=True"
+    "DefaultConnection": "Server=localhost\\developer;Initial Catalog=BlazorHero.CleanArchitecture;Integrated Security=True;MultipleActiveResultSets=True"
```
Comment (unchase) [inline src/Server/appsettings.Development.json:3]: Need to revert this change

File: src/Client/Resources/Shared/Components/LanguageSelector.ru.resx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    <value>Итальянский</value>
+  </data>
+  <data name="Khmer" xml:space="preserve">
+    <value>Кхмерский</value>
+  </data>
+  <data name="Russian" xml:space="preserve">
+    <value>русский</value>
+  </data>
+  <data name="Spanish" xml:space="preserve">
+    <value>испанский</value>
```
Comment (unchase) [inline src/Client/Resources/Shared/Components/LanguageSelector.ru.resx:142]: Should use the camel case

File: src/Server/appsettings.Development.json (hunk 1 above)
Comment (Artillio) [reply src/Server/appsettings.Development.json:3]: Yes Thanks!
Sorry
